### ISSUE TITLE
The installation of mmrotate will install an additional mmcv.

### ISSUE THREAD

**Issue by liuyanyi** [open-mmlab/mmrotate#6]
**Describe the bug**
The installation of mmrotate will install an additional mmcv although I have mmcv-full installed. 

The output of the pip list command is as follows:

```none
mmcv                1.4.5
mmcv-full           1.4.5
mmdet               2.21.0
mmrotate            0.1.0     
```

**Reproduction**

1. What command or script did you run?

```none
pip install -v -e .
```

**Bug fix**

Compared with other mmlab project, i think the difference may appeare in `requirements/runtime.txt`, maybe remove mmcv in runtime.txt can fix it.

**Comment by zytx121** [comment]
This is because only the latest version of mmcv (1.4.5)  contains all the operators required by MMRotate.

**Comment by liuyanyi** [comment]
Thanks for your reply, I follow the installation guide in a new conda environment, there are still two mmcv in pip list. Both training and testing work fine.
But on mmcv's document, it says both versions in the same environment may encounter errors, should I uninstall one of them?

**Comment by liufeinuaa** [comment]
I also have this problem, bug training and working fine....

**Comment by liufeinuaa** [comment]
I was tried modified the `requirements/runtime.txt`, change the 'mmcv' to 'mmcv-full', can fix this problem. 
In my conda env just one mmcv-full package, and run demo was fine.

**Comment by ZwwWayne** [comment]
Hi @liufeinuaa ,
Thanks for your solution. Would you like to create a PR to fix that?

**Comment by liufeinuaa** [comment]
Of course 

> Hi @liufeinuaa , Thanks for your solution. Would you like to create a PR to fix that?

**Comment by ZwwWayne** [comment]
resolved in #17
